src/libraries/Microsoft.Extensions.Hosting/src/HostOptions.cs

```diff
@@ -46,6 +62,13 @@ internal void Initialize(IConfiguration configuration)
                 ShutdownTimeout = TimeSpan.FromSeconds(seconds);
             }
 
+            timeoutSeconds = configuration["startupTimeoutSeconds"];
```

Comment:
Why seconds not TimeSpan?

Reply:
For consistency with shutdownTimeoutSeconds

Reply:
For consistency with "shutdownTimeoutSeconds" - see the lines just above this.